### PR TITLE
Start of creating the configuration process for an algorithm

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/config.jsp [hunk 1]
```diff
@@ -33,13 +33,175 @@
 				"id" : "gov.usgs.cida.gdp.wps.algorithm.FeatureCoverageIntersectionAlgorithm",
 				"name" : "FeatureCoverageIntersectionAlgorithm",
 				"title" : "WCS Subset",
-				"abstract" : "This service returns the subset of data that intersects a set of vector polygon features and a Web Coverage Service (WCS) data source. A GeoTIFF file will be returned."
+				"abstract" : "This service returns the subset of data that intersects a set of vector polygon features and a Web Coverage Service (WCS) data source. A GeoTIFF file will be returned.",
```
Comment (isuftin) [inline src/main/webapp/WEB-INF/jsp/config.jsp]: I've laid out two algorithms here. We can go back and query the WPS server or continue building out in this fashion. This is quicker for now and gives us some test data to go on.

File: src/main/webapp/js/advanced/views/AlgorithmConfigView.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/*jslint browser: true*/
+/*global Backbone*/
+var GDP = GDP || {};
+
+GDP.view = GDP.view || {};
+
+GDP.view.AlgorithmConfigView = Backbone.View.extend({
+	render : function () {
+		"use strict";
+		this.$el.html(this.template(this.collection.models));
```
Comment (isuftin) [inline src/main/webapp/js/advanced/views/AlgorithmConfigView.js]: Not sure what I'm doing here yet but merging this doesn't affect anything

File: src/main/webapp/js/advanced/views/ProcessView.js [hunk 3]
```diff
@@ -12,6 +15,9 @@ GDP.view.ProcessView = GDP.util.BaseView.extend({
 	render : function () {
 		"use strict";
 		this.$el.html(this.template(this.collection.models));
+		this.algorithmConfigView.$el = this.$("#container-process-configuration");
```
Comment (isuftin) [inline src/main/webapp/js/advanced/views/ProcessView.js]: Getting how to work with child views from http://spin.atomicobject.com/2014/01/25/render-nested-views-backbonejs/

Going to give it a try

**Comment by isuftin** [comment]
Found a lil bug

File: src/main/webapp/js/advanced/views/ProcessView.js [hunk 4]
```diff
@@ -5,6 +5,9 @@ var GDP = GDP || {};
 GDP.view = GDP.view || {};
```
Comment (cschroed-usgs) [inline src/main/webapp/js/advanced/views/ProcessView.js]: Are we namespacing with GDP.thing or GDP.ADVANCED.thing? I don't care which we use, but we should be consistent.

Comment (isuftin) [inline src/main/webapp/js/advanced/views/ProcessView.js]: I think .ADVANCED. is where we're going for now.

Comment (isuftin) [inline src/main/webapp/js/advanced/views/ProcessView.js]: I will change it on my end :D

Comment (isuftin) [inline src/main/webapp/js/advanced/views/ProcessView.js]: I'm going to walk back on this. Looks like AdvancedRouter is not using ADVANCED for any views (even though theyre in the advanced folder).

**Pull request closed without merging**